integration-intygstjanst/src/main/java/se/inera/intyg/minaintyg/integration/intygstjanst/CertificateStatusService.java

```diff
@@ -18,14 +18,15 @@ public List<CertificateStatusType> get(List<CertificateRelationDTO> relations,
       LocalDateTime issued) {
 
     final var statuses = getStatusesFromRelations(relations);
+
+    if (isReplaced(statuses)) {
+      return getListOfStatuses(statuses);
+    }
+
```

Comment:
Consider if adding an "if-statement", surrounding the sent & new if not replaced, more clearly state the intention of the code instead of returning if isReplaced?